server/user_oauth1.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package main
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"net/http"
+
+	"github.com/dghubble/oauth1"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+const paramOTS = "ots"
```

Comment:
This isn't used anywhere. Is it still needed?

Reply:
This was in prep for a future ticket, not used yet and I'll remove it for now. What tool did you use to catch this?

Reply:
https://golangci.com/ :wink: